src/views/RecIM/components/Forms/WaiverForm/index.jsx

```diff
@@ -53,8 +53,11 @@ const WaiverForm = ({ username, createSnackbar, openWaiverForm, setOpenWaiverFor
 
   let waiverContent = (
     <Typography margin={4}>
+      <Typography variant="h5">
+        Note: This form is currently not legally binding, a paper form will be provided in the
+        meantime. Please contact a Rec-IM staff to sign up
+      </Typography>
```

Comment:
```suggestion
      <Typography variant="h5">
        Please note: This acknowledgement is just a formality, you will need to sign a physical
        waiver before competing. Please contact
        <a href='mailto:rec.im@gordon.edu'>Rec.IM@gordon.edu</a>
        to sign up.
      </Typography>
```
I think this is a little bit better wording. Also it's good to give them a specific way to sign the paper waiver because otherwise, it's super hard to figure out who is in charge of Rec/IM and get their email.